Makefile

```diff
@@ -90,13 +82,13 @@ birdcl:
 	chmod +x birdcl
 
 ## Run the UTs in a container.
-ut: calico_test/.calico_test.created
+ut:
 	docker run --rm -v `pwd`/calicoctl:/code calico/test \
 		nosetests $(UT_TO_RUN) -c nose.cfg
 	docker run --rm -v `pwd`/calico_node:/code calico/test \
 		nosetests filesystem/tests --with-coverage --cover-package=filesystem
 
-ut-circle: calico_test/.calico_test.created dist/calicoctl
+ut-circle: dist/calicoctl
```

Comment:
Circle is configured to just run `make ut`. So the tests that this PR triggered show that it should not be a problem.

However, this merge will cause failed dockerhub builds of calico/test, as its configured to point to this repository. My next task is to reinitialize that repo, pointing it at libcalico instead